### PR TITLE
Dont fail completely on Internet Explorer

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

**Comment by skjnldsv** [comment]
You should take the oppotrunity to upgrade your babel config 
https://github.com/nextcloud/notifications/blob/master/.babelrc.js
`babel.config.js`

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module.exports = {
+	plugins: [
+		'@babel/plugin-syntax-dynamic-import',
+		['@babel/plugin-proposal-class-properties', { loose: true }],
```
Comment (skjnldsv) [inline babel.config.js:4]:
```suggestion
```
You're not using the class so you don't need this one :)

Same for the dynamic loading, you're not using async javascript loading :)

File: babel.config.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	],
```
Comment (skjnldsv) [inline babel.config.js:5]:
```suggestion
```
You're not using the class so you don't need this one :)

Same for the dynamic loading, you're not using async javascript loading :)

Comment (nickvergessen) [reply babel.config.js]: woopsy, gone

File: babel.config.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	presets: [
+		[
+			'@babel/preset-env',
+			{
+				corejs: 3,
+				useBuiltIns: 'entry',
```
Comment (skjnldsv) [inline babel.config.js:7]: Perfect, entry will take care of the import then :)

File: .eslintrc.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
 	extends: [
-		'nextcloud'
+		'@nextcloud'
```
Comment (skjnldsv) [inline .eslintrc.js:3]: oups :see_no_evil:

**Pull request merged**